dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/log/remote/AbsRemoteLogHandler.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.common.log.remote;
+
+import org.apache.dolphinscheduler.common.constants.Constants;
+import org.apache.dolphinscheduler.common.utils.PropertyUtils;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.Closeable;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import lombok.extern.slf4j.Slf4j;
+
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.azure.storage.blob.specialized.BlobInputStream;
+import com.azure.storage.common.StorageSharedKeyCredential;
+
+@Slf4j
+public class AbsRemoteLogHandler implements RemoteLogHandler, Closeable {
+
+    private String accountName;
+
+    private String accountKey;
+
+    private String containerName;
+
+    private BlobContainerClient blobContainerClient;
+
+    private static AbsRemoteLogHandler instance;
+
+    private AbsRemoteLogHandler() {
+
+    }
+
+    public static synchronized AbsRemoteLogHandler getInstance() {
+        if (instance == null) {
+            instance = new AbsRemoteLogHandler();
+            instance.init();
+        }
+
+        return instance;
+    }
+
+    public void init() {
+        accountName = readAccountName();
+        accountKey = readAccountKey();
+        containerName = readContainerName();
+        blobContainerClient = buildBlobContainerClient();
+    }
```

Comment:
Can we move this into the constructor?
When we get the instance by 
```
AbsRemoteLogHandler.getInstance();
```
We don't need to execute `init` since the instance has already been initialized.

Reply:
Indeed this is a solution.
Currently it follows the patterns with [GcsRemoteLogHandler , s3remotelogger and etc.](https://github.com/apache/dolphinscheduler/blob/a72505c926225989cbfdebf726a99fc1068b0b3d/dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/log/remote/GcsRemoteLogHandler.java)
I will create another PR for fixing this.

Reply:
I create an PR for fixing this https://github.com/apache/dolphinscheduler/pull/15780